desktop/e2e/run.js

```diff
@@ -116,7 +118,7 @@ async function run() {
 		await videoRecorder.startVideo();
 
 		const parentEnv = process.env;
-		app = spawnDetached( CWD, SPAWN_CMD, APP_ARGS, null, {
+		app = spawnDetached( CWD, SPAWN_CMD, APP_ARGS, appLog.fd, {
```

Comment:
Not a big deal but just curious about the need to set the file descriptor here? Today in Circle the Linux build saves the app log as an artifact without this change ([example](https://app.circleci.com/pipelines/github/Automattic/wp-calypso/80454/workflows/77433aec-271b-4798-a2ca-692781c3ed03/jobs/949084/artifacts)) - is this due to different in Linux environment?

Reply:
Correct, this was mostly to diagnose CI problems, in particular Chrome failing to start. We can redirect it to stdout (so it is captured in TC logs) or to its own file